databases.go

```diff
@@ -229,3 +231,22 @@ func (c *Client) DatabaseDisableFeature(ctx context.Context, app, addonID, featu
 
 	return res, nil
 }
+
+type MaintenanceWindowParams struct {
+	WeekdayUTC      *int `json:"weekday_utc,omitempty"`
+	StartingHourUTC *int `json:"starting_hour_utc,omitempty"`
+}
+
+func (c *Client) DatabaseUpdateMaintenanceWindow(ctx context.Context, app, addonID string, params MaintenanceWindowParams) (Database, error) {
+	var dbRes DatabaseRes
+	err := c.DBAPI(app, addonID).ResourceUpdate(ctx, "databases", addonID, map[string]interface{}{
+		"database": map[string]interface{}{
+			"maintenance_window": params,
+		},
+	}, &dbRes)
+
+	if err != nil {
+		return Database{}, errgo.Notef(err, "fail to update maintenance window")
+	}
+	return dbRes.Database, nil
+}
```

Comment:
code organization question: I would move this into the `maintenance.go` file, what do you think? It seems more in line with other features, such as backup